vividus-extension-selenium/src/main/java/org/vividus/selenium/manager/GenericWebDriverManager.java

```diff
@@ -63,6 +66,12 @@ public Dimension getSize()
         return screenSize;
     }
 
+    @Subscribe
+    public void onWebDriverQuit(AfterWebDriverQuitEvent event)
+    {
+        webDriverManagerContext.reset();
```

Comment:
the old reset (https://github.com/vividus-framework/vividus/blob/master/vividus-extension-selenium/src/main/java/org/vividus/steps/WebDriverSetupSteps.java#L81) should be removed in this case